Packs/Maltiverse/Integrations/Maltiverse/Maltiverse_test.py

```diff
@@ -58,3 +59,15 @@ def test_file(requests_mock):
     _, outputs, _ = file_command(client, args)
 
     assert outputs == EXPECTED_FILE_RESULT
+
+
+def test_file_no_process_list(requests_mock):
```

Comment:
please add a docstring with given-when-then
```suggestion
def test_file_command__missing_process_list_field_in_response(requests_mock):
```

Reply:
done